pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -1074,6 +1202,15 @@ protected void RunFunctionalTest(string defaultUiMode = UiModes.PROTEOMIC)
                 }
             }
 
+            try
+            {
+                _shotManager.SaveToFile();
+            }
+            catch (Exception e)     //TODO: it probably shouldn't interfere with the test like this, but this exception should be logged somehow.

```

Comment:
It might be better to not catch this exception, and let it get handled by the test framework as a failure.
The only reason to catch it would be if you were afraid that you might be hiding some exception that happened earlier in the test.

Reply:
This code is removed, see above.